frontend/src/pages/signup/types/signupInfo.ts

```diff
@@ -0,0 +1,7 @@
+export interface SignupInfo {
+  loginId: string;
+  name: string;
+  gender: '남' | '여';
```

Comment:
p4: 타입을 따로 빼는 건 어때? SignupInfo에서 꺼내서 쓸 일이 있는 거 같아! 


```suggestion
type Gender = '남' | '여'

...

  gender: Gender;
```

Reply:
[refactor: SignupInfo 의 gender 타입 분리](https://github.com/woowacourse-teams/2025-Fit-toring/pull/274/commits/5ea8c1d8746dd57e0c0edbc872ac1908ef96f3c9)